design/archive/nab_status_update-alternative.md

```diff
@@ -0,0 +1,87 @@
+# Developer Workflow: NonAdminBackup Phase and Conditions within NonAdminBackup Status
+
+## Version
+
+This is `alternative` of the design document. Simplifier approach is described in the [active design] doc.
+
+## Overview
+
+This document outlines the design around updating NonAdminBackup objects Phase and Conditions within Status.
+
+### Phase
+
+A NonAdminBackup's `status` field has a `phase` field, which is updated by NAC controller.
+
+The `phase` is a simple one high-level summary of the lifecycle of an NonAdminBackup.
+
+It is always a one well defined value, that is intended to be a comprehensive state of a NonAdminBackup.
+
+Those are are the possible values for phase:
+
+| **Value** | **Description**                 |
```

Comment:
Lets add a table where we map the `Phase` and `Conditions` as well as `VeleroBackupPhase` with each other for easier understanding

Reply:
That's why we have diagram which is integral part of that .md file with Phase/Condition within workflow. Do we need a table to support that diagram as well? Attaching screenshot of the diagram in case you don's see it in the review 'view'

![image](https://github.com/migtools/oadp-non-admin/assets/16115240/5de62c0f-887b-484c-a767-a5f8d1f1be8a)

Reply:
I saw the diagram :) 
ok lets at least tabulate `velero phases` with `NAB phases` ?

Reply:
on the gray squares, should not ALL of them have Queued populated :question: